README.md

```diff
@@ -88,44 +94,54 @@ Once integration test is completed, developer can delete the setup using `TearDo
 <details>
 <summary><b>Example</b></summary>
 
+## example_test.go
+
 ```go
 
 package example
 
 import (
+	"fmt"
 	"testing"
 	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/openebs/CITF"
-	"github.com/openebs/CITF/utils/k8s"
+	citfoptions "github.com/openebs/CITF/citf_options"
 )
 
+var CitfInstance citf.CITF
+
 func TestIntegrationExample(t *testing.T) {
 	RegisterFailHandler(Fail)
+
+	var err error
+	// Initializing CITF without config file.
+	// Also We should not include K8S as currently we don't have kubernetes environment setup
+	CitfInstance, err = citf.NewCITF(citfoptions.CreateOptionsIncludeAllButK8s(""))
+	Expect(err).NotTo(HaveOccurred())
+
 	RunSpecs(t, "Integration Test Suite")
 }
 
-// Initializing CITF without config file.
-var CitfInstance, err = citf.NewCITF("")
-
 var _ = BeforeSuite(func() {
 
 	// Setting up the default Platform i.e minikube
-	err = CitfInstance.Environment.Setup()
+	err := CitfInstance.Environment.Setup()
 	Expect(err).NotTo(HaveOccurred())
 
-	// if you are using minikube version greater than 0.24.1
-	// then you have to update the K8s config
-	// We're working to remove this step in upcoming changes.
-	CitfInstance.K8S, err = k8s.NewK8S()
+	// You have to update the K8s config when environment has been set up
+	// this extra step will be unsolicited in upcoming changes.
+	CitfInstance.Reload(citfoptions.CreateOptionsIncludeAll(""))
 	Expect(err).NotTo(HaveOccurred())
 
+	fmt.Printf("clientset: %+v\n", CitfInstance.K8S.Clientset)
```

Comment:
Is it required ?

Reply:
else everything looks good to me !!

Reply:
No, removing it now. I put it for debug.
Nice catch